app/assets/javascripts/code/services/_heat.js

```diff
@@ -93,6 +93,21 @@ export const heat = ($rootScope, params, storage) => {
       }
     },
 
+    levelName: function(value){
+      if (value < this.getValue("lowest")) {
+        return null;
+      } else if (value < this.getValue("low")) {
+        return "low";
+      } else if (value < this.getValue("mid")) {
+        return "mid";
+      } else if (value < this.getValue("high")) {
+        return "high";
+      } else if (value < this.getValue("highest")) {
+        return "highest";
+      } else {
+        return null;
+      }
+    },
```

Comment:
this returns the strings 'low', 'mid' ... cause css classes are called like this. But maybe it shouldn't be coupled like that. The classes could be named green, orange etc...